RadixWallet/Features/SettingsFeature/SecurityFactors/SecurityFactors+View.swift

```diff
@@ -0,0 +1,143 @@
+private typealias S = L10n.SecurityFactors
+
+extension SecurityFactors.State {
+	var viewState: SecurityFactors.ViewState {
+		.init(
+			seedPhrasesCount: seedPhrasesCount,
+			ledgerWalletsCount: ledgerWalletsCount,
+			isSeedPhraseRequiredToRecoverAccounts: isSeedPhraseRequiredToRecoverAccounts
+		)
+	}
+}
+
+// MARK: - SecurityFactors.View
+
+public extension SecurityFactors {
+	struct ViewState: Equatable {
+		let seedPhrasesCount: Int?
+		let ledgerWalletsCount: Int?
+		let isSeedPhraseRequiredToRecoverAccounts: Bool
+	}
+
+	@MainActor
+	struct View: SwiftUI.View {
+		private let store: StoreOf<SecurityFactors>
+
+		public init(store: StoreOf<SecurityFactors>) {
+			self.store = store
+		}
+
+		public var body: some SwiftUI.View {
+			content
+				.setUpNavigationBar(title: S.title)
+				.tint(.app.gray1)
+				.foregroundColor(.app.gray1)
+				.presentsLoadingViewOverlay()
+				.destinations(with: store)
+		}
+	}
+}
+
+@MainActor
+private extension SecurityFactors.View {
+	var content: some View {
+		WithViewStore(store, observe: \.viewState, send: { .view($0) }) { viewStore in
+			ScrollView {
+				VStack(spacing: .zero) {
+					ForEach(rows(viewStore: viewStore)) { kind in
+						kind.build(viewStore: viewStore)
+					}
+				}
+			}
+			.background(Color.app.gray4)
+			.onFirstTask { @MainActor in
+				await viewStore.send(.onFirstTask).finish()
+			}
+		}
+	}
+
+	func rows(viewStore: ViewStoreOf<SecurityFactors>) -> [SettingsRow<SecurityFactors>] {
+		[
+			.header(S.subtitle),
+			.model(
+				title: S.SeedPhrases.title,
+				subtitle: S.SeedPhrases.subtitle,
+				detail: seedPhrasesDetail(viewStore),
+				hints: seedPhraseHints(viewStore),
+				icon: .asset(AssetResource.seedPhrases),
+				action: .seedPhrasesButtonTapped
+			),
+			.model(
+				title: S.LedgerWallet.title,
+				subtitle: S.LedgerWallet.subtitle,
+				detail: ledgerWalletsDetail(viewStore),
+				icon: .asset(AssetResource.ledger),
+				action: .ledgerWalletsButtonTapped
+			),
+		]
+	}
+
+	func seedPhrasesDetail(_ viewStore: ViewStoreOf<SecurityFactors>) -> String? {
```

Comment:
As @kugel3 also mentioned, there is really no point as it seems to passing the viewStore, you could just pass the propert, like here, it seems that just passing `seedPhrasesCount` to this function should be enough.

You may want to pass the viewStore if you want to send actions, which doesn't seem to be the case.